src/ApiService/ApiService/onefuzzlib/Queue.cs

```diff
@@ -16,12 +17,12 @@ public interface IQueue
 public class Queue : IQueue
 {
     IStorage _storage;
-    ILog _logger;
+    ILogger _logger;
 
-    public Queue(IStorage storage, ILog logger)
+    public Queue(IStorage storage, ILoggerFactory loggerFactory)
```

Comment:
here it should be ILogTraceFactory. 

The factory is set as a member in the object. When a new function triggered on the object - create a new Log intance from the factory and set CorrelationId on it (we could move this into middleware, I don't know how dependency injection will work with middleware though...)